Listener/ExceptionListener.php

```diff
@@ -40,9 +40,14 @@ public static function getSubscribedEvents(): array
 
     public function onKernelException(GetResponseForExceptionEvent $event): void
```

Comment:
this class does not exists anymore in 5.0 .

I'm trying to fix all this changes in #239